src/hotspot/share/jfr/leakprofiler/chains/bfsClosure.hpp

```diff
@@ -28,17 +28,20 @@
 #include "jfr/leakprofiler/utilities/unifiedOopRef.hpp"
 #include "memory/iterator.hpp"
 
-class BitSet;
 class Edge;
 class EdgeStore;
 class EdgeQueue;
+template<MEMFLAGS F>
+class ObjectBitSet;
+
+typedef ObjectBitSet<mtTracing> JFRBitSet;
```

Comment:
Can you include memory/allocation.hpp for the MEMFLAGS? iterator.hpp already does, but better to be include-clean

Reply:
Optional: move this typedef to an own jfr/leakprofiler/jfrbitset.hpp and include that one instead of objectbitset.inline.hpp in the callee cpp files